gatsby-node.js

```diff
@@ -300,6 +300,38 @@ exports.sourceNodes = async ({
     };
 
     await createNode(nodeReleases);
+
+    activity.end();
```

Comment:
`sourceNodes` becoming quite big and hard-to-read. I'd recommend to split it to several function calls or event modules

Reply:
> `sourceNodes` becoming quite big and hard-to-read. I'd recommend to split it to several function calls or event modules

I agree. Do you want this done in this PR or a separate refactoring PR? Im leaning towards separate so unit tests can be added.

Reply:
I think this can be done in another PR